src/doc/src/reference/specifying-dependencies.md

```diff
@@ -254,30 +298,50 @@ hello_utils = { path = "hello_utils" }
 ```
 
 This tells Cargo that we depend on a crate called `hello_utils` which is found
-in the `hello_utils` folder (relative to the `Cargo.toml` it’s written in).
+in the `hello_utils` folder, relative to the `Cargo.toml` file it’s written in.
+
+The next `cargo build` will automatically build `hello_utils` and
+all of its dependencies.
+
+#### No local path traversal
```

Comment:
ditto, and the heading is a bit hard to understand at first glance, as people not even know git dependencies traverse the repo. Yet I don't have any better suggestion.

```suggestion
### No local path traversal
```